src/tests/pages/orders.page.js

```diff
@@ -0,0 +1,204 @@
+        this.natureEndTimeLocator = this.page.locator("#nature-end_time");
+        this.natureEndFoodLocator = this.page.locator("#nature-end_food");
+
+        this.toastErrorLocator = this.page.locator(".toast-error");
+        this.toastMessage = this.page.locator(".toast-message");
+        this.fieldErrorLocator = this.page.locator(".invalid-feedback");
+        this.orderConfirmationTitle = this.page.getByRole('heading', { name: 'Děkujeme za objednávku' });
+
+        // Group form fields:
+        this.orderFormFields = {
+            icoField: this.icoFieldLocator,
+            clientField: this.clientFieldLocator,
+            addressField: this.addressFieldLocator,
+            substituteField: this.substituteFieldLocator,
+            contactNameField: this.contactNameFieldLocator,
+            contactPhoneField: this.contactPhoneFieldLocator,
+            contactEmailField: this.contactEmailFieldLocator,
+            startDate1Field: this.startDate1FieldLocator,
+            endDate1Field: this.endDate1FieldLocator,
+            startDate2Field: this.startDate2FieldLocator,
+            endDate2Field: this.endDate2FieldLocator,
+            startDate3Field: this.startDate3FieldLocator,
+            endDate3Field: this.endDate3FieldLocator
+        };
+
+        // Group camp information form fields:
+        this.campOrderFields = {
+            campDatePartField: this.campDatePartLocator,
+            campStudentsNumberField: this.campStudentsNumberLocator,
+            campStudentsAgeField: this.campStudentsAgeLocator,
+            campAdultsNumberField: this.campAdultsNumberLocator
+        };
+
+        // Group school in nature information form fields:
+        this.natureOrderFields = {
+            natureStudentsNumberField: this.natureStudentsNumberLocator,
+            natureStudentsAgeField: this.natureStudentsAgeLocator,
+            natureAdultsNumberField: this.natureAdultsNumberLocator,
+            natureStartTimeField: this.natureStartTimeLocator,
+            natureStartFoodField: this.natureStartFoodLocator,
+            natureEndTimeField: this.natureEndTimeLocator,
+            natureEndFoodField: this.natureEndFoodLocator
+        };
+    }
+
+    async open() {
+        await this.page.goto("/objednavka/pridat");
+    }
+
+    async fillClientInfo(
+        ico = icoNumber,
+        client = clientName,
+        address = clientAddress,
+        substituteText = substitute,
+        contactNameText = contactName,
+        contactPhoneText = contactPhone,
+        contactEmailText = contactEmail
+    ) {
+        await this.icoFieldLocator.fill(ico);
+        await this.substituteFieldLocator.fill(substituteText);
+        await this.contactNameFieldLocator.fill(contactNameText);
+        await this.contactPhoneFieldLocator.fill(contactPhoneText);
+        await this.contactEmailFieldLocator.fill(contactEmailText);
+
+        if (this.clientFieldLocator.inputValue() !== client) {
```

Comment:
_inputValue()_ vrací _Promise_
```suggestion
        if (await this.clientFieldLocator.inputValue() !== client) {
```